Rakefile

```diff
@@ -1,7 +1,7 @@
 require "json"
 require "yaml"
 
-DEFAULT_BASE_OS = "ubuntu-15.10"
+DEFAULT_BASE_OS = "ubuntu-15.04"
```

Comment:
Yes :sob: I had a long debugging session via screen sharing with @bascht yesterday. The result was that there was something really fucked up in the network preferences of the box. I had the gut feeling that there might be something wrong in the 15.10 version, and that gut feeling was right: I created the box with 15.10: Could not connect to it, network was fucked up. I ONLY changed this line, generated a new base.json and everything worked.